docs/Configuration.md

```diff
@@ -1753,7 +1754,7 @@ test('does not show prototypes for object and array inline', () => {
 
 Default: `undefined`
 
-The path to a module that can resolve test<->snapshot path. This config option lets you customize where Jest stores snapshot files on disk.
+The path to a module that can resolve test\<->snapshot path. This config option lets you customize where Jest stores snapshot files on disk.
 
```

Comment:
In MDX v2 this does not compile. Escaping is a good solution.

Reply:
@SimenB this can't be extracted as is to a docs prep PR because on MDX v1, `\` will stay visible. The only solution that works on MDX v1 + v2 is `test&lt;->snapshot` which IMHO is much less maintainable compared to `\<->`.

For this reason I think it's better to keep this change in this PR